### PR TITLE
ZCS-816 Added README

### PR DESCRIPTION
**Pull request by gordyt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    sudo yum install java-1.8.0-openjdk java-1.8.0-openjdk-devel ruby git cpan wget
+    # install specific perl modules
+    sudo cpan IPC::Cmd
+    cd /tmp
+    # install maven
+    wget http://mirror.metrocast.net/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+    sudo tar -xf apache-maven-3.3.9-bin.tar.gz
+    sudo mv apache-maven-3.3.9 /opt
+    sudo vi /etc/profile.d/maven.sh
+    export PATH=/opt/apache-maven-3.3.9/bin:$PATH
```
Comment (plobbes) [inline README.md]: for safety (embedded spaces) always use double quotes w/export

File: README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    # install current version of ant
+    wget https://www.apache.org/dist/ant/binaries/apache-ant-1.9.9-bin.zip
+    sudo unzip apache-ant-1.9.9-bin.zip
+    sudo mv apache-ant-1.9.9 /opt
+    sudo vi /etc/profile.d/ant.sh
+    export PATH=/opt/apache-ant-1.9.9/bin:$PATH
```
Comment (plobbes) [inline README.md]: for safety (embedded spaces) always use double quotes w/export

File: README.md (hunk 1 above)
Comment (plobbes) [inline README.md]: Note: we could make this a one-liner and avoid vi (same applies below):
`echo 'export PATH="/opt/apache-maven-3.3.9/bin:$PATH"' | sudo tee -a /etc/profile.d/maven.sh`

**Comment by gordyt** [comment]
Update PR to address issues that Phil noted.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    # install maven
+    wget http://mirror.metrocast.net/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+    sudo tar -xf apache-maven-3.3.9-bin.tar.gz
+    sudo mv apache-maven-3.3.9 /opt
+    echo 'export PATH="/opt/apache-maven-3.3.9/bin:$PATH"' | sudo tee -a /etc/profile.d/maven.sh
+    # install current version of ant
+    wget https://www.apache.org/dist/ant/binaries/apache-ant-1.9.9-bin.zip
+    sudo unzip apache-ant-1.9.9-bin.zip
+    sudo mv apache-ant-1.9.9 /opt
+    sudo vi /etc/profile.d/ant.sh
```
Comment (plobbes) [inline README.md]: avoid vi here using tee just like line 63 above

Comment (gordyt) [inline README.md]: Thanks Phil!  It's been updated.

**Pull request merged**